server_admin/topics/groups.adoc

```diff
@@ -31,6 +31,7 @@ image:{project_images}/user-groups.png[]
 Select a group from the `Available Groups` tree and hit the `join` button to add the user to a group.  Vice versa to remove a group.
 Here we've added the user _Jim_ to the _North America_ sales group.  If you go back to the detail page for that group and
 select the `Membership` tab, _Jim_ is now displayed there.
+User is notified via email every time he is added to/ removed from a group.
```

Comment:
Avoid latin words such as "via" and gender-based words such as "he"

Suggested:  The user is notified by email when that user is added to or removed from a group.